protected/humhub/services/MetaSearchService.php

```diff
@@ -97,6 +107,16 @@ public function getTotal(): int
         return isset($this->totalCount) ? (int) $this->totalCount : 0;
     }
 
+    /**
+     * Get total number results as short integer number
+     *
+     * @return string
+     */
+    public function getShortTotal(): string
```

Comment:
Why don't we use the formatter directly here?

Reply:
@luke- Do you mean to remove the merhod `getShortTotal()` and keep only the meyhod `getTotal()` with new code like this?

```php
public function getTotal(): string
{
    return Yii::$app->formatter->asShortInteger($this->totalCount);
}
```

Really I thought to keep the method `getTotal()` which returns now only the value from the protected property `$this->totalCount` for some external access case in future. But if you think we don't need it I can remove the `getShortTotal()`, let me know.

Reply:
@yurabakhtin I thought we could directly use the method 
`Yii::$app->formatter->asShortInteger($this->totalCount)` instead of a extra method. 
 But it's not that important. We can leave it as it is.